app/uk/gov/hmrc/cdsimportsdds/util/RESTFormatters.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsdds.util
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.cdsimportsdds.models.ImportsDeclaration
+
+object RESTFormatters {
```

Comment:
won't need this if we could have `format` defined in companion objects for model classes

Reply:
I want to keep the model free from technology specific code as per the ports and adapters pattern (aka hexagonal architecture). We could move the formatters to their respective ports i.e. the REST ones would live in the controllers inbound port and the Mongo ones would live in the repositories outbound port